src/modules/stackoverflow/stack.js

```diff
@@ -0,0 +1,83 @@
+import React,{useState,useEffect} from "react";
+import { useSelector,useDispatch } from 'react-redux';
+import { loadTopics,sortDesc,sortAsc } from "./stack.actions";
+
+const tableStyles = {
+    width:'200px',
+    border:'1px solid grey',
+}
+
+const Stack = () => {
+    const topicsList = useSelector(store=>store.stack.topics);
+    const [inputTopic, setInputTopic] = useState('');
+    const [sortType, setSortType] = useState('')
+    const dispatch = useDispatch();
+    console.log(topicsList);
+    console.log(sortType);
+
+    const handleShowTopic = (e)=>{
+        e.preventDefault();
+        dispatch(loadTopics(inputTopic));
+        setInputTopic('');
+    }
+
+    useEffect(() => {
+        if(sortType === 'desc'){
+            dispatch(sortDesc())
+        }
+        else if(sortType === 'asc') {
+            dispatch(sortAsc())
+        }
+    },[sortType])
+
+    const renderTopicsList = () => {
+        return topicsList.map(item => {
+            return (
+                <>
+                    <tr key={item.question_id}>
+                        <td>{item.owner.display_name}</td>
+                        <td>{item.owner.reputation}</td>
+                        <td>{item.creation_date}</td>
```

Comment:
Tak zdecydowanie lepie można zaprezentować te dane.
Tutaj np. znajdziesz rozwiązanie: https://stackoverflow.com/questions/847185/convert-a-unix-timestamp-to-time-in-javascript